content/uk/gallery1/index.md

```diff
@@ -0,0 +1,66 @@
+---
+author: "Hugo Authors"
+title: "4-6 listopada. Vistavka"
+type: gallery
+---
+
+<div class='text-left mt-8 mx-auto container px-6 text-justify mb-20'>
```

Comment:
Each page for the gallery we can name by numbers like /uk/gallery/1, /uk/gallery/2...